server/db.mjs

```diff
@@ -33,6 +33,13 @@ export const getPlants = async (zone) =>
     [zone],
   );
 
+export const addPlant = async (container) => {
+  await db.any("INSERT INTO containers (user_id, plant_id) VALUES ($1, $2);", [
+    container.userid,
+    container.plantid,
+  ])[0];
```

Comment:
What does the `[0]` mean here? Does it return the entire new row back?

Reply:
I believe it's the index position since db.any returns an array, i just want the first record returned